plugins/modules/appliance_access_dcui.py

```diff
@@ -70,11 +70,13 @@
         type: bool
 author:
 - Ansible Cloud Team (@ansible-collections)
-version_added: 2.0.0
+version_added: 1.0.0
```

Comment:
This actually must be changed in all modules back to `version_added: 1.0.0`

Reply:
all of the modules are set to `1.0.0` in the PR, did you mean change it back to `2.0.0`?

Reply:
ive restored the original version values, and added a playbook task to do that automatically next time